assets/javascripts/discourse/components/discourse-post-event/index.gjs

```diff
@@ -53,8 +54,11 @@ export default class DiscoursePostEvent extends Component {
   }
 
   get eventName() {
-    return emojiUnescape(
-      escapeExpression(this.args.event.name) || this.args.event.post.topic.title
+    return htmlSafe(
+      emojiUnescape(
+        escapeExpression(this.args.event.name) ||
+          this.args.event.post.topic.title
+      )
     );
```

Comment:
We need to escape `topic.title` as well.

If we want to keep this all manual, then I think we should split it up like:
```suggestion
    const unsafeRawTitle = this.args.event.name || this.args.event.post.topic.title;
    const escapedTitle = escapeExpression(unsafeRawTitle);
    const escapedTitleWithEmoji = emojiUnescape(escapedTitle);
    return htmlSafe(escapedTitleWithEmoji);
```

---

Or, even better, we can do the emoji escaping using our hbs helper which has all the escaping built-in.

So make the getter a simple thing like:

```
get eventName(){
  return this.args.event.name || this.args.event.post.topic.title;
}
```

and then import/use the helper:
...
```gjs
import replaceEmoji from "discourse/helpers/replace-emoji";
...
<template>
  {{replaceEmoji this.eventName}}
</template>
```